src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
I don't think you want to create a translation here, because it may already exist. There's another action defined that will select a translation.